olp-cpp-sdk-core/src/http/ios/OLPHttpTask.mm

```diff
@@ -143,10 +143,10 @@ - (BOOL)cancel {
 - (void)didCompleteWithError:(NSError*)error {
   OLP_SDK_LOG_DEBUG_F(kLogTag,
                       "Task is completed, request_id=%llu, url=%s, "
-                      "task_id=%u, error=%i",
+                      "task_id=%u, status=%i, error=%i",
                       self.requestId, [self.url UTF8String],
                       (unsigned int)self.dataTask.taskIdentifier,
-                      (int)error.code);
+                      self.responseData.status, (int)error.code);
```

Comment:
Is this the HTTP status code or the error code from the response? We need the HTTP status code!